tlv/fuzz_test.go

```diff
@@ -97,6 +97,13 @@ func FuzzVarBytes(f *testing.F) {
 	})
 }
 
+func FuzzBool(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte) {
+		var val []byte
```

Comment:
`val` needs to be a `bool`, otherwise this won't go past the first `if` in `EBool`. The rest looks good!

Reply:
Thanks all done